1.1.0/jessie/build-projectjson/README.md

```diff
@@ -11,8 +11,10 @@ This repository contains images that are used to compile/publish ASP.NET Core ap
 
 - [`1.1.0-projectjson`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1.0/jessie/build-projectjson/Dockerfile)
```

Comment:
I wish every Docker README.md would sort in descending order however this is not the norm.  What is the reason for being different?